src/syscall/syscall_libc.go

```diff
@@ -60,6 +60,5 @@ func splitSlice(p []byte) (buf *byte, len uintptr) {
 	return slice.buf, slice.len
 }
 
-// ssize_t write(int fd, const void *buf, size_t count)
```

Comment:
Nit: you probably didn't intend to remove this line?

Reply:
Oh, yeah. I thought I added it. Let me rollback that.